### ISSUE TITLE
They added bosses?

### ISSUE THREAD

**Issue by giv13** [meepen/salien-bot#156]
Look at this: https://www.youtube.com/watch?v=xt3UOnAv90o. They added bosses?

Does your script support this? Or do I need to search for the boss manually? Thanks!

![2018-06-30_22-34-16](https://user-images.githubusercontent.com/18557389/42127652-2284d700-7cb6-11e8-9b25-c1b7aab4fdc3.png)

**Comment by meepen** [comment]
Should support it

**Comment by Makaka2** [comment]
Yes, it supports boss rounds. I tried a new feature this morning.

**Comment by Makaka2** [comment]
Though it's little unstable. I hope @meepen will fix it soon.

**Comment by meepen** [comment]
Can you make a bug report about how it's unstable as well as your version

**Comment by darkl1ch** [comment]
well. the headless mode didnt do anything on the boss planet. :(

**Comment by Makaka2** [comment]
Ok. I'm using the latest version, _headless_, re-downloaded it just 1 hour ago. Firstly, the percentage of the boss HP is broken. Secondly, when the fight ends, game freezing (but scores are counting anyway) and to continue you must reload the script.

**Comment by csis0247** [comment]
I can second the report by @Makaka2 . Just fought a boss on an easy tile on Time Manipulation Planet. Percentage HP was like 0.XX% all the way, and headless failed to move on to a new zone after the fight but scores of the boss fight were recorded. Regrettably I forgot to add the --log switch after updating my script version to 2.1.5, so I can't provide you with logs to debug.

**Comment by Makaka2** [comment]
@darkl1ch You must use a special command besides primary. Add an "-e" supplement to usual "node headless" command.

**Comment by ahakola** [comment]
I had the newest version at the time of the boss fight and my headless also stopped picking new zones after boss fight ended. It didn't even show the updated score. I waited few minutes (At least 5mins) and it was still looping without picking new zone or showing the new score. After restarting the app it showed the updated score and picked new zone.

**Comment by Spartanul** [comment]
In my case, whenever there is a boss fight, I must manually accept the battle as this script does not do it automatically. Am I missing something? :(

**Comment by csis0247** [comment]
@darkl1ch you should update the script because boss-support is enabled by default without having to specify `-e` since commit [11f7e73fae266d8c0a495307c032eef075ef42ec] (https://github.com/meepen/salien-bot/commit/11f7e73fae266d8c0a495307c032eef075ef42ec) 17 hours ago.

**Comment by ahakola** [comment]
@Spartanul if you are running the headless, maybe you didn't give your script enough time to notice the fight or the servers were hammered with join requests from other users and couldn't get through before you manually zoned to the fight?

**Comment by Spartanul** [comment]
@ahakola - I'm just running the simple "Userscript Mode".

**Comment by ahakola** [comment]
@Makaka2 probably serverside issues, that might be also why there was hickups after the boss fight.

**Comment by darkl1ch** [comment]
thx for the advices, but i had the latest version which i could, but during the boss level, the "headless" bot didnt do anything.

**Comment by JohnBacardi** [comment]
I've been checking https://wrya.net/tmp/salien-status/all.html , yet so far I haven't seen any red triangle that represents the Boss battle. Add it to the bot so it automatically starts the correct planet with the correct "Boss-battle" for maximum XP?

**Comment by ahakola** [comment]
That hung up after the boss fight is probably caused by error in the network.js #162

**Comment by JohnBacardi** [comment]
Just fought a Boss, gaining 2.5 million XP
I joined https://discordapp.com/channels/459435127932715008/462403694122631169
Boss is added manually!!
This script helped me, but at the end it didn't press the button, but I did in time before the script relaunched the game!

**Comment by callmenemo491** [comment]
Am running: index.user.js (version 29),  i dont see the script currently supporting the new skill that appears during the boss battle.

**Comment by hptwo** [comment]
I'm using index.user.js version 29.  I've been manually clicking on the 'Fight!' button to start the boss fight.  I have also been manually hitting the '6' key on my keyboard to activate the healing ability.  I'm looking through the attacks.js file in the developer console to try and kludge together a HealAttack like FreezeAttack and try it out the next time I encounter a boss.

**Comment by meepen** [comment]
thanks @hptwo feel free to make a pr to add that! the issues with headless are fixed.
